docs/src/main/sphinx/admin/properties-query-management.rst

```diff
@@ -181,6 +199,25 @@ Timeout value for remote tasks that fail to communicate with the
 coordinator. If the coordinator is unable to receive updates from a remote task
 before this value is reached, the coordinator treats the task as failed.
 
+``query.remote-task.max-request-size``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** :ref:`prop-type-data-size`
+* **Default value:** ``8``
```

Comment:
thats not a valid data size value

Reply:
Yeah, this should be 8MB, and you don't need to specify that it's in megabytes. A data size is `2GB` or `16MB` or `10KB`, the user specifies their own units.